src/onefetch/cli.rs

```diff
@@ -220,17 +222,37 @@ impl Cli {
         .arg(
             Arg::with_name("off")
                 .long("off")
-                .help("Only shows the info lines.")
-                .conflicts_with_all(&["image", "ascii-language", "ascii-input"]),
-            ).get_matches();
+                .help("Only shows the info lines. DEPRECATED: use \"--hide-logo true\" instead.")
+                .conflicts_with_all(&["image", "ascii-language", "ascii-input", "hide-logo"]),
+            )
+        .arg(
+            Arg::with_name("hide-logo")
+                .long("hide-logo")
+                .value_name("auto | true | false")
+                .takes_value(true)
+                .max_values(1)
+                .multiple(false)
+                .possible_values(&["auto", "true", "false"])
+                .help("Will hide the logo if true. If set to auto, the logo will be hidden if the terminal size is too small. If set to false, the logo will be shown no matter what.")
+                .conflicts_with("off")
+        ).get_matches();
 
         let no_bold = matches.is_present("no-bold");
         let no_merges = matches.is_present("no-merge-commits");
         let no_color_palette = matches.is_present("no-color-palette");
         let print_languages = matches.is_present("languages");
         let print_package_managers = matches.is_present("package-managers");
-        let art_off = matches.is_present("off");
+        let mut art_off = matches.is_present("off");
         let true_color = cli_utils::is_truecolor_terminal();
+        let max_term_size = 95;
+
+        // The --off flag was passed in
+        if art_off {
+            std::println!(
```

Comment:
Just curious: were you having an issue with `println!` (no `std::`)?

Reply:
I don't know any other way to do it so..